pyt/core/module_definitions.py

```diff
@@ -4,7 +4,8 @@
 import ast
 
 
-# Contains all project definitions for a program run:
+# Contains all project definitions for a program run
+# Only used in framework_adaptor.py, but modified here
 project_definitions = dict()
```

Comment:
My only issue with that is [that it has a return value](https://www.tutorialspoint.com/python/dictionary_setdefault.htm) that we don't use, I feel like it would be nicer if we were using the return value. I did enjoy using [a defaultdict in definition_chains.py](https://github.com/python-security/pyt/blob/f7d9ff3c5413fcea771048e0115bd26adc20421c/pyt/analysis/definition_chains.py#L20) though.